src/components/file-input/file-input-test.stories.mdx

```diff
@@ -0,0 +1,54 @@
+import { Meta, Story, Preview } from "@storybook/addon-docs";
```

Comment:
```suggestion
import { Meta, Story, Canvas } from "@storybook/addon-docs";
```

Can you change this please to be in line with: https://github.com/Sage/carbon/pull/4595